test/rules/namespace/disallow/test.ts.lint

```diff
@@ -0,0 +1,17 @@
+namespace Foo {
```

Comment:
This file should be in `test/rules/namespace/default/` to indicate that it's a test for the default configuration